src/app/components/Image/index.tsx

```diff
@@ -66,17 +65,21 @@ const Image = ({
   const [isLoaded, setIsLoaded] = useState(false);
   const showPlaceholder = placeholder && !isLoaded;
   const hasDimensions = width && height;
-  const [aspectRatioX, aspectRatioY] =
-    (aspectRatio && aspectRatio) ||
-    (hasDimensions && [width, height]) ||
-    DEFAULT_ASPECT_RATIO;
+  const hasFixedAspectRatio = !!aspectRatio || !!hasDimensions;
+
+  // [16, 9] fallback here is to stop a typescript error
+  const [aspectRatioX, aspectRatioY] = aspectRatio ||
+    (hasDimensions && [width, height]) || [16, 9];
```

Comment:
```suggestion
  const [aspectRatioX, aspectRatioY] = aspectRatio ||
    (hasDimensions && [width, height]) || [null, null];
```
I've been wracking my brains to see if I can find a way to make this unecessary but I can't quite work it out 🤔 This suggestion may be a bit clearer as it doesn't imply any logical behaviour in the event both conditions aren't fullfiled; width and height will be set to null - leaving it at 16/9 gives the impression a default is still in place.

Reply:
Ah glad to hear I wasn't the only one puzzled by it. I can do this, though I'll need to one of the below to stop Typescript errors.

One is cleaner but the other is more bulletproof

```
  const legacyBrowserAspectRatio = getLegacyBrowserAspectRatio(
    aspectRatioX as number,
    aspectRatioY as number,
  );
```
or 
```
const getLegacyBrowserAspectRatio = (x: number | null, y: number | null) => {
  if (x === null || y === null) return 0;
  return roundNumber((y / x) * 100)
    .toString()
    .concat('%');
};
```

Reply:
Decided to go for the former - just seems simpler and it should always be true

Reply:
I'd probably go for the first one, I'm not convinced we've got to the bottom of this; I think you could structure the code differently to satisfy the compiler but I've wasted too mich time thinking about that now 😅

Reply:
Agree...